site/book/02-concepts/03-functions.md

```diff
@@ -0,0 +1,66 @@
+A kpt function (also called a _KRM function_) is a containerized program that can perform CRUD
+operations on KRM resources stored on the local filesystem. kpt functions are the extensible
+mechanism to automate mutation and validation of KRM resources. Some example use cases:
+
+- Enforce all `Namespace` resources to have a `cost-center` label.
+- Add a label to resources based on some filtering criteria
+- Use a `Team` custom resource to generate a `Namespace` and associated organization-mandated
+  defaults (e.g. `RBAC`, `ResourceQuota`, etc.) when bootstraping a new team
+- Bulk transformation of all `PodSecurityPolicy` resources to improve the security posture.
+
+Since functions are containerized, they can encapsulate different toolchains, languages, and
+runtimes. For example, the function container image can encapsulate:
+
+- A Go binary built using kpt's official library
+- Wrap an existing KRM tool such as `kubeval`
+- Invoke a bash script performing low-level operations
+- The interpreter for "executable configuration" such as `Starlark` or `Rego`
+
+To astute readers, this model will sound familiar: functions are the client-side analog to
+Kubernetes controllers: </br>
```

Comment:
</br> is the HTML tag there to ensure special rendering?

Reply:
Yes, the table is rendered too close to text otherwise.